ui/src/containers/SampleViewContainer.js

```diff
@@ -96,7 +96,7 @@ class SampleViewContainer extends Component {
         <PhaseInput
           phase={this.props.sampleViewState.currentPhase}
           phaseList={this.props.sampleViewState.phaseList}
-          sendPhase={this.props.sampleViewActions.sendCurrentPhase}
+          sendPhase={this.props.sampleViewActions.changeCurrentPhase}
```

Comment:
What do you think about changing `sendPhase` to `changePhase` as well  ?

Reply:
I noticed it when I worked on #1136  :wink:

Reply:
Oh yes, I noticed just now :)